api/app/signals/apps/signals/migrations/0040_SIG-1014.py

```diff
@@ -0,0 +1,110 @@
+        ('Handhaving op afval', ['ASC', 'THOR', ], 'A3DMC', ),
+    ],
+    # Main
+    'Wegen, verkeer, straatmeubilair': [
+        # Sub, departments
+        ('Autom. Verzinkbare palen', ['V&OR - VIS', ], 'A3WMC', ),
+        ('Bewegwijzering', ['V&OR - VIS', ], 'A3WEC', ),
+        ('Camerasystemen', ['V&OR - VIS', ], 'A3WEC', ),
+        ('Lichthinder', ['V&OR OVL', ], 'STOPEC', ),
+        ('Parkeer verwijssysteem', ['V&OR - VIS', ], 'A3WEC', ),
+        ('Stadsplattegronden', ['V&OR - VIS', ], 'A3WEC', ),
+        ('Verdeelkasten / bekabeling', ['V&OR - VIS', ], 'A3WEC', ),
+        ('Verlichting netstoring', ['V&OR OVL', ], 'STOPEC', ),
+    ],
+    # Main
+    'Ondermijning': [
+        # Sub, departments, handling
+        ('Vermoeden', ['Openbare Orde & Veiligheid', ], 'ONDERMIJNING', ),
+    ],
+    # Main
+    'Overig': [
+        # Sub, departments, handling
+        ('Overige dienstverlening', ['CCA', 'ASC', ], 'REST', ),
+    ]
+}
+
+
+def forward_categories(apps, schema_editor):
+    category_model = apps.get_model('signals', 'Category')
+    departments_model = apps.get_model('signals', 'Department')
+
+    # Create the new departments
+    departments_model.objects.create(name='V&OR - VIS', code='VIS')
+    department_oov = departments_model.objects.create(name='Openbare Orde & Veiligheid', code='OOV')
+
+    # Create the new "main" categories
+    category_ondermijning = category_model.objects.create(name='Ondermijning',
+                                                          slug=slugify('Ondermijning'),
+                                                          handling='ONDERMIJNING')
+    category_ondermijning.departments.add(department_oov)
+    category_ondermijning.save()
+
+    # Create the new "sub" categories
+    for parent, children in add.items():
+        parent_category = category_model.objects.get(name=parent, parent__isnull=True)
+
+        for child in children:
+            child_category = category_model.objects.create(name=child[0],
+                                                           slug=slugify(child[0]),
+                                                           parent=parent_category,
+                                                           handling=child[2])
+
+            departments = departments_model.objects.filter(name__in=child[1])
+            if departments.exists():
+                for department in departments:
+                    child_category.departments.add(department)
+                child_category.save()
+
+    # We only soft delete the following categories
+    category_model.objects.filter(
+        name__in=[
+            'Vuurwerkoverlast',
+            'Personen op het water',
+        ],
+        parent__name='Overlast van en door personen of groepen'
+    ).update(is_active=False)
+
+    category_model.objects.filter(
+        name='Deelfiets',
+        parent__name='Overlast in de openbare ruimte'
+    ).update(is_active=False)
+
```

Comment:
Change the "Auto / Scooter ..." category link to department as requested as well.